Backend/Backend/Services/EpisodeService.cs

```diff
@@ -43,8 +69,12 @@ public class EpisodeService : IEpisodeService
             {
                 throw new Exception("Uploading file not Successfull"); 
             }
+
             episode.EpisodeName = createEpisodeRequest.EpisodeName!;
             episode.IsExplicit = createEpisodeRequest.IsExplicit!;
+            episode.ReleaseDate = DateTime.Now;
```

Comment:
we should be using UTC here, and everywhere else for that matter. This will cause issues with time in the future